src/styles/blocks_header/_header.scss

```
@@ -0,0 +1,15 @@
+@import "variables";
```

I doubt variables actually import - they're not in the same folder, right?
so i guess it should be
```suggestion
@import "../variables";
```
or i'm wrong - let me know